controllers/operands/tto.go

```diff
@@ -0,0 +1,117 @@
+package operands
+
+import (
+	"errors"
+	"reflect"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	ttov1alpha1 "github.com/kubevirt/tekton-tasks-operator/api/v1alpha1"
+
+	hcov1beta1 "github.com/kubevirt/hyperconverged-cluster-operator/api/v1beta1"
+	"github.com/kubevirt/hyperconverged-cluster-operator/controllers/common"
+	"github.com/kubevirt/hyperconverged-cluster-operator/pkg/util"
+	hcoutil "github.com/kubevirt/hyperconverged-cluster-operator/pkg/util"
+)
+
+type ttoHandler struct {
+	genericOperand
+}
```

Comment:
usually we don't embed the generic handler, but just adding an alias. Please consider this.
```suggestion
type ttoHandler genericOperand
```